### ISSUE TITLE
Web app starter look and feel breaking if i am using polymer 2.5.0 

### ISSUE THREAD

**Issue by sravanmca21** [PredixDev/predix-webapp-starter#30]
I am trying to update polymer 2.5.0 in bower.json but components are not getting loading when i change this version. Please let me know if there is a web app starter which supports polymer 2 or higher.

**Comment by gstroup** [comment]
No, at this time the webapp-starter does not support Polymer 2.  There is a "px-sample-app" which supports Polymer 2, but does not include all the server features in the webapp-starter.

https://github.com/predixdesignsystem/px-sample-app
